src/dist/invgamma.rs

```diff
@@ -264,9 +264,28 @@ macro_rules! impl_traits {
 
         impl Sampleable<$kind> for InvGamma {
             fn draw<R: Rng>(&self, rng: &mut R) -> $kind {
+                const MAX_TRIES: usize = 10;
```

Comment:
Is 10 enough?

Reply:
Not sure. I was a little worried making it too big, since it could suddenly slow down for no obvious reason